plugins/modules/katello_repository.py

```diff
@@ -120,43 +120,28 @@
     gpg_key: RPM-GPG-KEY-my-product2
 '''
 
-RETURN = '''# '''
+RETURN = ''' # '''
 
-try:
-    from nailgun.entities import (
-        Repository,
-    )
-
-    from ansible.module_utils.ansible_nailgun_cement import (
-        find_organization,
-        find_product,
-        find_content_credential,
-        find_repository,
-        naildown_entity_state,
-        sanitize_entity_dict,
-    )
-except ImportError:
-    pass
 
-from ansible.module_utils.foreman_helper import KatelloEntityAnsibleModule
+from ansible.module_utils.foreman_helper import KatelloEntityApypieAnsibleModule
 
 
 # This is the only true source for names (and conversions thereof)
 name_map = {
```

Comment:
I wonder why there is no `organization(_id)` in the `name_map` here

Reply:
Or is it because a Product already has an Org and thus it gets the id via that route?

Reply:
Good question. I guess, because the repository is scoped by the product, which in turn is scoped by the organization. The tests passing suggest that it is sufficient.